.github/workflows/run-integration-tests.yaml

```
@@ -105,6 +101,18 @@ jobs:
           echo "version tag: ${{ env.VERSION_TAG }}"
           make catalog-build 
           make catalog-push
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)